OxfordBodleian/Juel-Jensen/BDLaethg35.xml

```diff
@@ -213,8 +213,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         </item>
                                         </list>
                                 </collation>
-                                <condition key="good">The manuscript folios are in good condition and show only slight traces of use.
-                                  The sewing has become undone since the sewing thread broke in many points. Both boards are detached from the bookblock, together with <ref target="#q1"/>.</condition>
+                                <condition key="good">The manuscript folios show only slight traces of use.
```

Comment:
Is "manuscript folios" somehow redundant? Perhaps "the bookblock" or "the manuscript"?